theano/gpuarray/tests/test_abstractconv.py

```diff
@@ -196,7 +196,7 @@ def setup_class(cls):
         cls.shared = staticmethod(gpuarray_shared_constructor)
         cls.mode = mode_with_gpu.excluding('cudnn')
 
-    def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1, 1)):
+    def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1, 1), u=False):
```

Comment:
Actually, that would require passing the parameter here, which I haven't done. Rather than add it here, I'll just put a SkipTest right here.